gui/app/modes/CommandPressureMode.qml

```diff
@@ -6,17 +6,18 @@ import "../controls"
 import ".."
 
 Mode {
+    id: mode
```

Comment:
I think the TODO below is obsolete

Reply:
I guess updating the ScopeView with a Timer on the QML and leaking GUI details (XYSeries) inside the GUI state side is not ideal, IMHO.

I don't have the ideal solution in mind, but I think the better way, for now, is to have `GUIStateContainer` to hold a `QVector<QPointF>` property for each series with the latest update, and send a property update signal. We do the binding on the QML side.

Also how often the series is updated should be a GUIStateContainer detail. On the same way how often the `ScopeView` will draw the lastest serie is a UI detail.

What do you think?

Reply:
This is fine with me. I don't remember why I did it the way it is now.